lib/videojs-playlist-ui.js

```diff
@@ -66,6 +66,7 @@ const createThumbnail = function(thumbnail) {
     // simple thumbnails
     let img = document.createElement('img');
     img.src = thumbnail;
+    img.alt = '';
```

Comment:
Oh, right. We should be using `aria-labelledby` in that case. But probably should go with whichever is easier. I don't think an empty string is correct behavior.